akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -1185,7 +1185,11 @@ class Http2ServerSpec extends AkkaSpecWithMaterializer("""
 
       "reject other frame than HEADERS/PUSH_PROMISE in idle state with connection-level PROTOCOL_ERROR (5.1)" inAssertAllStagesStopped new SimpleRequestResponseRoundtripSetup {
         network.sendDATA(9, endStream = true, HPackSpecExamples.C41FirstRequestWithHuffman)
-        network.expectGOAWAY()
+        val (lastStreamId, error) = network.expectGOAWAY()
+        // We are over-estimating the last stream for which 'data may have been processed' here.
+        // That is allowed. Returning '0' would be better here, since the stream is not actually processed.
```

Comment:
Is that something we should improve?

Reply:
It doesn't seem very high-priority, but it would be nice. I'll create a ticket.